internal/verifier/recheck_test.go

```diff
@@ -332,17 +445,36 @@ func (suite *IntegrationTestSuite) TestLargeIDInsertions() {
 		SourceDocumentCount: 1,
 		SourceByteCount:     types.ByteCount(overlyLarge),
 		FirstMismatchTime:   map[int32]bson.DateTime{},
+		SrcTimestamp:        option.Some(bson.Timestamp{123, 0}),
 	}
 
 	t2 := t1
 	t2.Ids = mslices.Of(mbson.ToRawValue(id2))
+	t2.SrcTimestamp = option.Some(bson.Timestamp{123, 1})
 
 	t3 := t1
 	t3.Ids = mslices.Of(mbson.ToRawValue(id3))
+	t3.SrcTimestamp = option.Some(bson.Timestamp{123, 2})
 
 	suite.ElementsMatch([]VerificationTask{t1, t2, t3}, foundTasks)
 }
 
+func fetchVerifierCurrentTasks(
+	ctx context.Context,
+	t *testing.T,
+	verifier *Verifier,
+) []VerificationTask {
+	taskColl := verifier.metaClient.Database(verifier.metaDBName).Collection(verificationTasksCollection)
+
+	cursor, err := taskColl.Find(ctx, bson.D{{"generation", verifier.generation}})
+	require.NoError(t, err)
+	var actualTasks []VerificationTask
```

Comment:
```suggestion

	var actualTasks []VerificationTask
```
nit: I would appreciate a new line for readability.